frontend/src/components/Main.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+
+// Components
+import { SignUp } from './SignUp';
+import { LogIn } from './LogIn';
+import { ErrorMessage } from './ErrorMessage';
+import { Content } from './Content';
+
+// ----------------------------------------------------------------
+
```

Comment:
Love that you added a specific component for handling whether the user is logged in/not 🙌